### PR TITLE
Define generalized PMI validator

### PR DESCRIPTION
**Pull request by marcoscaceres**
Needed for https://github.com/w3c/payment-request/issues/464

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/validate.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/44e8787...ff09155.html)

### CODE REVIEW

**Review by rsolomakhin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -72,6 +72,27 @@
...
+          Validity
+        </h3>
+        <p>
+          The steps to <dfn>validate a payment method identifier</dfn> with a
+          <var>pmi</var> string are given by the following algorithm. It
+          returns either true or false.
+        </p>
+        <ol>
+          <li>Let <var>url</var> be the result of running the <a>basic URL
+          parser</a> with <var>pmi</var>.
```
Comment (domenic) [inline index.html]: basic URL parser link isn't working in https://rawgit.com/w3c/payment-method-id/97fcb88628ab8a34c25d89b1caf435abf12d1e47/index.html

File: index.html [hunk 2]
```diff
@@ -72,6 +72,27 @@
           <a>standardized payment method identifier</a>.
         </li>
       </ul>
+      <section>
+        <h3>
+          Validity
+        </h3>
+        <p>
+          The steps to <dfn>validate a payment method identifier</dfn> with a
+          <var>pmi</var> string are given by the following algorithm. It
```
Comment (domenic) [inline index.html]: Nit: both here and in other algorithms you seem to put the type after the variable name, but that's pretty unusual in web specs. "string _pmi_" and "URL _url_" would be more expected.

**Review by domenic — APPROVED** [review]
LGTM with nits

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html]: Makes sense, will fix here and then update globally. The other ones are badly written also, so will clean them up to match.

**Pull request merged**